### PR TITLE
bypass volume-driver validation

### PR DESCRIPTION
**Pull request by wlan0**
@ibuildthecloud

### CODE REVIEW

**Comment by ibuildthecloud** [comment]
almost-LGTM.  Everything looks good.  Since we don't have matrix builds, this code needs to tested against Docker 1.7/8/9 manually.  Once I do that I'll merge.

**Comment by wlan0** [comment]
@ibuildthecloud I can create a test suite to test against docker 1.7/1.8/1.9 etc. 

For some reason, this fix doesn't work with docker-1.8. Still investigating root cause.

File: cattle/plugins/docker/compute.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 from . import docker_client, pull_image
 from . import DockerConfig
-from . import DockerPool
+from .storage import DockerPool
```
Comment (cjellick) [inline cattle/plugins/docker/compute.py]: Why did this line need to cahnge?

**Comment by cjellick** [comment]
@wlan0 tested against docker 1.6-1.9
LGTM

**Pull request merged**